static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx

```diff
@@ -131,13 +148,33 @@ function DefaultGroupEventDetailsContent({
         />
       </StyledDataSection>
       {event.userReport && (
-        <EventDataSection title={t('User Feedback')} type="user-feedback">
-          <EventUserFeedback
-            report={event.userReport}
-            orgSlug={organization.slug}
-            issueId={group.id}
-            showEventLink={false}
-          />
+        <EventDataSection
+          title={t('User Feedback')}
+          type="user-feedback"
+          actions={
+            <ErrorBoundary mini>
+              <ButtonBar gap={1}>
+                <HideUserFeedbackButton
+                  isLoading={promptLoading}
+                  isError={promptError || isPromptDismissed === undefined}
+                  isHidden={isPromptDismissed !== undefined ? isPromptDismissed : false}
```

Comment:
```suggestion
                  isHidden={isPromptDismissed ?? false}
```

first part will automatically check for `undefined`